api/src/main/java/org/openmrs/module/patientflags/api/FlagService.java

```diff
@@ -168,12 +197,30 @@
 	/**
 	 * Retrieve a specific Tag based on its tagId
 	 * 
-	 * @param tagId
+	 * @param tagId The ID of the tag  to retrieve
 	 * @return the Tag which matches the specific ID
 	 */
 	@Transactional(readOnly = true)
 	@Authorized(value = { "Manage Flags", "Test Flags" }, requireAll = false)
 	public Tag getTag(Integer tagId);
+	/**
+	 * Retrieve a specific Tag based on its uuid
+	 *
+	 * @param uuid The uuid of the tag  to retrieve
+	 * @return the Tag which matches the specific uuid
+	 */
+	@Transactional(readOnly = true)
+	@Authorized(value = { "Manage Flags", "Test Flags" }, requireAll = false)
+	public Tag getTagByUuid(String uuid);

```

Comment:
You need to have at least a line between methods.

Reply:
Done